function/function.go

```diff
@@ -128,23 +128,23 @@ func (f *Function) callAWSLambda(payload []byte) ([]byte, error) {
 		config = config.WithCredentials(credentials.NewStaticCredentials(f.Provider.AWSAccessKeyID, f.Provider.AWSSecretAccessKey, f.Provider.AWSSessionToken))
 	}
 
-	awsSession, err := session.NewSession(config)
-	if err != nil {
-		return nil, &ErrFunctionError{err}
-	}
-	awslambda := lambda.New(awsSession)
+	awslambda := lambda.New(session.New(config))
 
 	invokeOutput, err := awslambda.Invoke(&lambda.InvokeInput{
 		FunctionName: &f.Provider.ARN,
 		Payload:      payload,
 	})
 	if err != nil {
-		if receivedAWSErr, ok := err.(awserr.Error); ok {
-			switch receivedAWSErr.Code() {
+		if awserr, ok := err.(awserr.Error); ok {
```

Comment:
`awserr` collides with an imported package

Reply:
If there is no automated check for that (with `gometalinter`) I would assume it's fine.

Reply:
My linter is reporting a warning on this. Not that it would be problematic, just that it is bad practice.

Reply:
👍